src/components/dialogs/generator-creation-dialog.js

```diff
@@ -0,0 +1,663 @@
+                                label={intl.formatMessage({
+                                    id: 'ActivePowerText',
+                                })}
+                                adornmentPosition={'end'}
+                                adornmentText={'MW'}
+                                value={activePowerSetpoint}
+                                onChange={handleChangeActivePowerSetpoint}
+                                defaultValue=""
+                                {...(errors.get('active-power-set-point')
+                                    ?.error && {
+                                    error: true,
+                                    helperText: intl.formatMessage({
+                                        id: errors.get('active-power-set-point')
+                                            ?.errorMsgId,
+                                    }),
+                                })}
+                            />
+                        </Grid>
+                        <Grid item xs={4} align="left">
+                            <TextFieldWithAdornment
+                                id="reactive-power-set-point"
+                                label={intl.formatMessage({
+                                    id: 'ReactivePowerText',
+                                })}
+                                adornmentPosition={'end'}
+                                adornmentText={'MVar'}
+                                disabled={!enabledReactivePowerSetpoint}
+                                value={reactivePowerSetpoint}
+                                onChange={handleChangeReactivePowerSetpoint}
+                                defaultValue=""
+                                {...(errors.get('reactive-power-set-point')
+                                    ?.error && {
+                                    error: true,
+                                    helperText: intl.formatMessage({
+                                        id: errors.get(
+                                            'reactive-power-set-point'
+                                        )?.errorMsgId,
+                                    }),
+                                })}
+                            />
+                        </Grid>
+                    </Grid>
+                    <Grid container spacing={2}>
+                        <Grid item xs={4} align="left">
+                            <FormControlLabel
+                                className={classes.voltageRegulation}
+                                id="voltage-regulation"
+                                control={
+                                    <Switch
+                                        checked={voltageRegulation}
+                                        onChange={(e) =>
+                                            handleChangeVoltageRegulation(e)
+                                        }
+                                        value="checked"
```

Comment:
What is the purpose of value for a Switch component?

Reply:
property removed